src/config.py

```diff
@@ -412,6 +412,17 @@ def log_dirs(self) -> str:
             [os.fspath(storage.location) for storage in self.charm.model.storages["data"]]
         )
 
+    @property
+    def rack_properties(self) -> List[str]:
+        """Builds all properties related to rack awareness configuration.
+
+        Returns:
+            List of properties to be set
+        """
+        # TODO: not sure if we should make this an instance attribute like the other paths
+        rack_path = f"{self.charm.snap.CONF_PATH}/rack.properties"
+        return safe_get_file(rack_path) or []
```

Comment:
**thought:** Personally I like it hard-coded here. As it's supposed to be an 'add-on', I like having the logic mostly contained in a single function.